docs/source/reference/configuration.md

```diff
@@ -35,6 +35,7 @@ These variables are optional that can be specified in the design configuration f
 |Variable|Description|
 |-|-|
 | `VERILOG_FILES_BLACKBOX` | Black-boxed, Verilog files where the implementation is ignored. Useful for pre-hardened macros you incorporate into your design, used during synthesis. |
+| `VERILOG_NETLISTS` | Verilog files of additional modules. Currently, they are used in multiconrer sta. Netlists must be powered. <br> (Default: NONE) |
```

Comment:
Typo: conrer. Also, how is this different from `VERILOG_FILES_BLACKBOX`?

Reply:
`VERILOG_FILES_BLACKBOX` is passed to yosys. Same files can't be passed to opensta since opensta only parses gatelevel netlists.

Reply:
Should we not unify? Why should a user have to present multiple Verilog views of the same thing?

VERILOG_FILES_BLACKBOX can also be gate-level netlists.

Reply:
But it can be rtl as well. In caravel we have simple_por which doesn't have a gl netlist. For that we read in the rtl as a blackbox module. @mo-hosni suggested that we might want to use VERILOG_NETLIST for yosys as well and then VERILOG_FILES_BLACKBOX + VERILOG_NETLISTS as read in by yosys to avoid duplicate files in both variables